as/plugins/org.jboss.ide.eclipse.as.jmx.integration/src/org/jboss/ide/eclipse/as/jmx/integration/AbstractJBossJMXConnectionProvider.java

```diff
@@ -43,71 +43,107 @@ public AbstractJBossJMXConnectionProvider() {
 	
```

Comment:
Changes to this file are about whether the jmx connection provided caches its connection for each server at what points and for how long. The two options are basically once-per-server-creation cache, or once-per-server-start cache.  The former is for jmx connections that will  always pull data from a runtime or a different model. The jolokia connection needs to be different any time the openshift server is started, though, so needs to be erased and a new one created after each stop of openshift server in case details in the connection need to be changed.